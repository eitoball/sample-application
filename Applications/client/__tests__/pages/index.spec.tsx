import { render, screen } from '@testing-library/react';
import Index from '../../src/pages/index';

test('Index', async () => {
   render(<Index />);

   const item = await screen.getByText('Hello, world!');
   expect(item).toBeTruthy();
   expect(item).toMatchSnapshot();
});
