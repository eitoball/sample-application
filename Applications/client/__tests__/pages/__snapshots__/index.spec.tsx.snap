// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index 1`] = `
<div>
  Hello, world!
</div>
`;
